ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -17,7 +17,7 @@ class Tag {
 export default class FormWorkerCreateWorkerLedComponent extends Component {
   // =services
   @service features;
-  @service browserObject;
+  @service('browser/window') window;
```

Comment:
Is this also supposed to update how we make references to these globals in desktop as well?

We still have references to the other way we access the globals such as in [this example](https://github.com/hashicorp/boundary-ui/blob/1f3a8ef135d1eadcfe6cc9428e80d32e1dbd223a/ui/desktop/app/routes/cluster-url.js#L31)

Reply:
@ZedLi yes good catch!